extras/static_code_analysis.sh

```diff
@@ -50,6 +50,9 @@ PEP8_IGNORES="E121,E123,E126,E226,E24,E704"
 PEP8_IGNORES_EXAMPLES="${PEP8_IGNORES},E402"
 PEP8_IGNORES_TOPO_MANUAL="${PEP8_IGNORES_EXAMPLES},E265"
 
+# Drop files that should not be checked (space-separated list).
+FILES_TO_IGNORE="libnestutil/sparsetable.h"
```

Comment:
I reviewed the files in following directories:
- conngen
- examples
- extras
- libnestutil

Everything looks fine. There is just one comment on the exclusion list, se eabove.